Documentation/k8s-pre-reqs.md

```diff
@@ -69,6 +69,61 @@ kubectl cluster-info
 
 Once you see a url response, your cluster is [ready for use by Rook](ceph-quickstart.md#deploy-rook).
 
+## Support for authenticated docker registries
+
+If you want to use an image from authenticated docker registry (e.g. for image cache/mirror), you'll need to 
+add an `imagePullSecret` to all relevant service accounts. This way all pods created by the operator (for service account: 
+`rook-ceph-system`) or all new pods in the namespace (for service account: `default`) will have the `imagePullSecret` added
+to their spec.
+
+The whole process is described in the [official kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+
+To get you started, here's a quick rundown for the ceph example from the [quickstart guide](/Documentation/ceph-quickstart.md).
+
+First, we'll create the secret for our registry as described [here](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod):
+
+```bash
+# for namespace rook-ceph-system (operator)
+kubectl -n rook-ceph-system create secret docker-registry my-registry-secret --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
+
+# and for namespace rook-ceph (cluster)
+kubectl -n rook-ceph create secret docker-registry my-registry-secret --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
+```
+
+Next we'll add the following snippet to all relevant service accounts as described [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account):
+
+```yaml
+imagePullSecrets:
+- name: my-registry-secret
+```
+
+The service accounts are:
```

Comment:
Setting the registry secret on the service accounts could also impact other storage providers such as minio, cockroachdb, and nfs. What if you add a subheading to make it more obvious this is an example of how to enable it on the ceph service accounts?

Reply:
Sorry, was out for a bit. Added a subheading as requested.